### PR TITLE
[blog] Adding introduction blog on KubeEdge - 2019-03-12-kubeedge-k8s-based-edge-intro.md

### PR DESCRIPTION
**Pull request by skdwriting**
Adding a blog post on Introduction to [KubeEdge](https://github.com/kubeedge/kubeedge), a Kubernetes based Edge computing platform. 
2019-03-12-kubeedge-k8s-based-edge-intro.md and related images.

Please note that KubeEdge team is also collaborating with Kubernetes IOT/EDGE WORKING GROUP. Also KubeEdge is one of the reference architecture discussed in the workgroup.

### CODE REVIEW

**Comment by m1093782566** [comment]
cc @stewart-yu

File: content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+
+Based on its roadmap and architecture, it tries to support all edge nodes, applications, devices and even the cluster management consistent with kuberenetes interface. This will help the edge cloud enact exactly like a cloud cluster. This can save a lot of time and cost on the edge cloud development deployment based on KubeEdge.
+
+KubeEdge provides a containerized edge computing platform, which is inherently scalable. As it's modular and optimized, it is light weight (66MB foot print and ~30MB running memory) and could be deployed on low resource devices. Similarly, the edge node can be of different hardware architecture and with different hardware configurations. For the device connectivity it can support multiple protocols (getting extended the list) and it uses a standard MQTT based communication. This helps in scaling the edge clusters with new nodes and devices efficiently. 
+
+
+**<p align="center">You heard it right!</p>**
+**<p align="center">KubeEdge Cloud Core modules are open sourced!</p>**
+
```
Comment (stewart-yu) [inline content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md]: what's thoes two line means?

File: content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+Even if want to add more control plane module based on the architecture refinement and improvement (for example enhanced security), it is simple as it uses consistent registration and modular communication within these modules.
+
+
+**<p align="center">KubeEdge provides scalable light weight Kubernetes Native Edge Computing Platform which can work in offline mode.</p>** 
+
+**<p align="center">It helps simplified edge application development and deployment.</p>**
+
+**<p align="center">Cloud vendor agnostic and can run the cloud core modules on any compute node.</p>**
+
```
Comment (stewart-yu) [inline content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md]: the same

File: content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+## Run Anywhere - Simple and Light
+
+As described, the KubeEdge Edge and Cloud core components can be deployed easily and can run the user applications. The edge core has a foot print of 66MB and just needs 30MB memory to run. Similarly the cloud core can run on any cloud nodes. (User can experience by running it on a laptop as well)
+
+The installation is simple and can be done in few steps:
+  1. Setup the pre-requisites Docker, Kubernetes, MQTT and openssl
+  2. Clone and Build KubeEdge Cloud and Edge
+  3. Run Cloud
+  4. Run Edge
+
```
Comment (stewart-yu) [inline content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md]: have format error, may be need blank line

File: content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+The detailed steps for each are available at https://github.com/kubeedge/kubeedge
+
```
Comment (stewart-yu) [inline content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md]: can we update link like: [KubeEdge/kubeedge](https://github.com/kubeedge/kubeedge) ?

**Comment by m1093782566** [comment]
@skdwriting Please sign CLA.

File: content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md (hunk 1 above)
Comment (skdwriting) [reply content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md]: Highlighting in between the blog these texts to call out that point. Aligned to center.

File: content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md (hunk 2 above)
Comment (skdwriting) [reply content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md]: Highlighting in between the blog these texts to call out that point. Aligned to center.

**Comment by skdwriting** [comment]
> Thanks for your PR @skdwriting 
> And need CLA
> /assign @kbarnard10 for review either

Signed the CLA. Thanks.

**Comment by m1093782566** [comment]
cc @cindyxing for IOT/Edge wg opinion.

File: content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md (hunk 3 above)
Comment (skdwriting) [reply content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md]: Sorry , could not make out the issue.

File: content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md (hunk 4 above)
Comment (skdwriting) [reply content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md]: Fixed.

File: content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md (hunk 3 above)
Comment (stewart-yu) [reply content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md]: image

Comment (stewart-yu) [reply content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md]: see the preview, please

**Comment by stewart-yu** [comment]
ping me back after all tech review done
hint: rebase the commit log more gracefully after all review done 
/assign

File: content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md (hunk 3 above)
Comment (skdwriting) [reply content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md]: kubeedge-blog-01

Somehow i see in the preview correctly.
However, inserted extra lines and ensure again. Kindly check, Thanks a lot for careful review.

**Comment by skdwriting** [comment]
@CindyXing, thanks for the comments. It helps.
Actually some of those points, like the extension of cloud, cost, etc were added in intro and also in the second section. However updated to call out clearer in the intro again. Please check.
Once you are fine, I will squash the commits, then the PR can be merged.

_Note: somehow, did not see the reply option to your comment, hence added as a new comment_

**Comment by CindyXing** [comment]
Done technical review. LGTM. Thanks.

**Comment by m1093782566** [comment]
@kbarnard10 @stewart-yu 

Per @CindyXing, we have done the technical review. Please give a final approval. Thanks!

**Comment by kbarnard10** [comment]
@m1093782566 Great - thanks! Will get this published first thing tomorrow.

**Comment by m1093782566** [comment]
@kbarnard10 

Cool! Thanks!

**Comment by stewart-yu** [comment]
/approve
@kbarnard10 @zacharysarah 
any others opinion? is it time to get it merged?

**Comment by skdwriting** [comment]
@stewart-yu  @kbarnard10 @zacharysarah @CindyXing , Thanks a lot.

**Pull request merged**